packages/panzer/mini-em/src/closures/MiniEM_TensorConductivity_impl.hpp

```diff
@@ -49,39 +49,40 @@ template <typename EvalT,typename Traits>
 void TensorConductivity<EvalT,Traits>::evaluateFields(typename Traits::EvalData workset)
 {
   using panzer::index_t;
-
+  auto conductivity_h = Kokkos::create_mirror_view(conductivity.get_static_view());
   if (ir_dim == 3) {
     for (index_t cell = 0; cell < workset.num_cells; ++cell) {
-      for (int point = 0; point < conductivity.extent_int(1); ++point) {
+      for (int point = 0; point < conductivity_h.extent_int(1); ++point) {
```

Comment:
Maybe I am missig somethign, but is there a reason not to run this code on device?